src/stamps/entries.js

```diff
@@ -32,13 +32,14 @@ const stamp = stampit()
 .methods({
   /**
    * Get all the content entries, based on the given parameters.
-   * **DO NOT** use several of id, alias and typeId at the same time - behaviour would be ungaranteed.
+   * **DO NOT** use several of id, alias, typeId and typeAlias at the same time - behaviour would be ungaranteed.
    * @param {object} [params] a parameters object
    * @param {boolean} [params.preview] when true, get the preview version
    * @param {string|date} [params.at] when given, get the version at the given time
    * @param {array} [params.id] an array of entry ids (strings)
    * @param {array} [params.alias] an array of entry aliases (strings)
    * @param {string} [params.typeId] only return entries matching this type id
```

Comment:
I see... I'll make it accept both a string (CSV or not) and an array, so that it's simpler when there's only one typeId and it doesn't introduce a breaking change.

Note I just checked if the typeAlias param behaves the same, and the doc seems to contradict itself on this:

> Get a list of Entries by passing in a list of Entry Type Aliases

> typeAlias An Entry Type Alias. Example: one-entry-type-alias.

Something to fix in the AppGrid doc maybe, @nitw-shobhit  ?